index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Ada Lovelace</title>
+    
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+
+  </head>
+  <body>
+
+    <nav>
+      <a href="#">About Me</a> | <a href="portfolio.html">Portfolio</a> | <a href="portrait.html">Fun Facts</a>
+
+    </nav>
+
+    <hr>
+
+    <h1> Ada Lovelace </h1>
+
+    <img src="800px-Ada_lovelace_portrait.jpg">
```

Comment:
Although not required, the best practice is to include the `alt` attribute which allows for accessibility to the visually impaired and is what appears if the resource fails to load.